### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/testing.md`

### PR DESCRIPTION
**Pull request by marcelomarkus**
- Portuguese translation
docs/pt/docs/tutorial/testing.md

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

**Review by cassiobotaro — APPROVED** [review]

File: docs/pt/docs/tutorial/testing.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+# Testando
+
+Graças ao <a href="https://www.starlette.io/testclient/" class="external-link" target="_blank">Starlette</a>, testar aplicativos **FastAPI** é fácil e agradável.
+
+Ele é baseado em <a href="https://www.python-httpx.org" class="external-link" target="_blank">HTTPX</a>, que por sua vez é projetado com base em Requests, por isso é muito familiar e intuitivo.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/testing.md:5]: I would change *"baseado em HTTPX"* to *"baseado **no** HTTPX"*

The same for requests: *"com base **no** Requests"*

It's just a suggestion ;)

File: docs/pt/docs/tutorial/testing.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+Com ele, você pode usar <a href="https://docs.pytest.org/" class="external-link" target="_blank">pytest</a> diretamente com **FastAPI**.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/testing.md:7]: I would add an *o* here:

*"você pode usar **o** pytest diretamente com **o** FastAPI"*

File: docs/pt/docs/tutorial/testing.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+
+## Usando `TestClient`
+
+/// info | "Informação"
+
+Para usar o `TestClient`, primeiro instale <a href="https://www.python-httpx.org" class="external-link" target="_blank">`httpx`</a>.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/testing.md:13]: I would add an **o** here:

*"primeiro instale **o** httpx"*

**Review by ceb10n — APPROVED** [review]
Just added some suggestions, but great work ✨

**Review by nazarepiedady — APPROVED** [review]

**Comment by alejsdev** [comment]
Thank you! @marcelomarkus :rocket:
 
And thanks to all for the reviews :sparkles:

**Pull request merged**